extras/kubernetes/heketi-bootstrap.json

```diff
@@ -42,6 +42,11 @@
         }
       },
       "spec": {
+        "selector": {
+          "matchLabels": {
+            "name": "deploy-heketi"
```

Comment:
I see that you have copied only one of the three labels in the template. Any particular reason for that?